src/contracts/core/DelegationManager.sol

```diff
@@ -742,11 +968,18 @@ contract DelegationManager is
         if (strategy == beaconChainETHStrategy) {
```

Comment:
if this function is this small and only called in one place, then let's just combine it where it's used

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/679/commits/a70ed62f13fa8be58c456c8d518ec71e7a8482ab